readme.md

```diff
@@ -108,18 +113,7 @@ json-stringify-safe:   deep          x 8,705 ops/sec ±0.87% (96 runs sampled)
 json-stringify-safe:   deep circular x 8,336 ops/sec ±2.20% (93 runs sampled)
 ```
 
-Comparing the deterministic `fast-safe-stringify` with known alternatives:
-(Running the `fast-json-stable-stringify` [benchmark][])
-
-```md
-fast-json-stable-stringify x 15,494 ops/sec ±1.59% (88 runs sampled)
-json-stable-stringify x 12,229 ops/sec ±1.32% (89 runs sampled)
-fast-stable-stringify x 16,226 ops/sec ±0.65% (92 runs sampled)
-faster-stable-stringify x 13,900 ops/sec ±1.05% (90 runs sampled)
-fast-safe-stringify x 26,528 ops/sec ±1.40% (91 runs sampled)
-
-The fastest is fast-safe-stringify
-```
+For comparison of the deterministic version check [`safe-stable-stringify`][].
```

Comment:
For stable stringify comparisons, see the [performance benchmarks](https://www.npmjs.com/package/safe-stable-stringify#performance--benchmarks) in the  [`safe-stable-stringify`][] readme.